src/core/colorspace.js

```diff
@@ -366,12 +368,10 @@ var ColorSpace = (function ColorSpaceClosure() {
           var range = params.getArray('Range');
           return ['LabCS', whitePoint, blackPoint, range];
         default:
-          error('unimplemented color space object "' + mode + '"');
+          throw new Error(`unimplemented color space object "${mode}"`);
```

Comment:
I'm not sure that I get why this isn't a `FormatError` instead, since this case looks quite similar to lines [number 289](https://github.com/mozilla/pdf.js/pull/8581/files#diff-b53da70ffcb20b79cde556d5176a903aR289) and [number 374](https://github.com/mozilla/pdf.js/pull/8581/files#diff-b53da70ffcb20b79cde556d5176a903aR374); am I missing something here?